Adding allocated and reserved memory values to memory timline view.

anupambhatnagar opened the pull request:
Summary: This diff adds the max allocated and max reserved memory values to the memory timeline plot.

Test Plan:
Executed

`buck run mode/dev-nosan kineto/libkineto/fb/integration_tests:pytorch_resnet_integration_test -- --enable_profiling --profile_memory --trace_handler=auto_trace --with_stack --record_shapes` on my devgpu.

The generated output is at
https://www.internalfb.com/manifold/explorer/ai_efficiency/tree/traces/dynocli/devgpu020.odn1.facebook.com/rank-0/rank-0.Aug_10_16_50_50.236946.pt.memorytl.html

 {F1067885545}
Screenshot of the html above
 {F1067886350}

Reviewed By: aaronenyeshi

Differential Revision: D48251791

aaronenyeshi reviewed (approved):
LGTM, reviewed internally!

anupambhatnagar committed:
Adding allocated and reserved memory values to memory timline view. (#107056)

Summary:
This diff adds the max allocated and max reserved memory values to the memory timeline plot.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/107056

Test Plan:
Executed

`buck run mode/dev-nosan kineto/libkineto/fb/integration_tests:pytorch_resnet_integration_test -- --enable_profiling --profile_memory --trace_handler=auto_trace --with_stack --record_shapes` on my devgpu.

The generated output is at
https://www.internalfb.com/manifold/explorer/ai_efficiency/tree/traces/dynocli/devgpu020.odn1.facebook.com/rank-0/rank-0.Aug_10_16_50_50.236946.pt.memorytl.html

 {F1067885545}
Screenshot of the html above
 {F1067886350}

Reviewed By: davidberard98

Differential Revision: D48251791

Pulled By: anupambhatnagar

fbshipit-source-id: 1eb1a7ea3b14f57c190ca47214758918aff7b504

aaronenyeshi reviewed (approved):
@pytorchbot merge

anupambhatnagar commented:
@pytorchbot merge
